[Bug]: Action "Project Creation" fails on main branch

### What happened?

A regression on main, it worked about a week ago

### Operating System

Linux

### What release version or branch did you use?

main

### Relevant log output

```shell
2024-07-04T08:37:28.7486711Z app/tests/integration/integration_test.py:43: AssertionError
2024-07-04T08:37:28.7487984Z ------------------------------ Captured log call -------------------------------
2024-07-04T08:37:28.7489114Z ERROR    grpc._common:_common.py:91 Exception serializing message!
2024-07-04T08:37:28.7490001Z Traceback (most recent call last):
2024-07-04T08:37:28.7491312Z   File "/home/vscode/.local/lib/python3.10/site-packages/grpc/_common.py", line 89, in _transform
2024-07-04T08:37:28.7492410Z     return transformer(message)
2024-07-04T08:37:28.7493947Z TypeError: descriptor 'SerializeToString' for 'google._upb._message.Message' objects doesn't apply to a 'list' object
2024-07-04T08:37:28.7495395Z =========================== short test summary info ============================
2024-07-04T08:37:28.7496585Z FAILED app/tests/integration/integration_test.py::test_set_position_not_allowed
2024-07-04T08:37:28.7497740Z !!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!
2024-07-04T08:37:28.7498676Z ============================== 1 failed in 21.11s ==============================
```
```


### Code of Conduct

- [X] I agree to follow this project's rules for contributions

--- a/.github/workflows/check-licenses.yml
+++ b/.github/workflows/check-licenses.yml
@@ -52,10 +52,12 @@ jobs:
 
       - name: Run dash
         shell: bash
-        continue-on-error: true
         run: |
           wget -O dash.jar "https://repo.eclipse.org/content/repositories/dash-licenses/org/eclipse/dash/org.eclipse.dash.licenses/1.0.2/org.eclipse.dash.licenses-1.0.2.jar"
-          java -jar dash.jar clearlydefined.input -summary DEPENDENCIES
+          java -jar dash.jar clearlydefined.input -summary DEPENDENCIES > dash.out 2>&1 || true
+          echo -e "Dash output: \n\`\`\` " >> $GITHUB_STEP_SUMMARY
+          cat dash.out >> $GITHUB_STEP_SUMMARY
+          echo -e "\n\`\`\`"
 
       - name: Upload dash input/output as artifacts
         uses: actions/upload-artifact@v4

--- a/.github/workflows/check-licenses.yml
+++ b/.github/workflows/check-licenses.yml
@@ -45,7 +45,7 @@ jobs:
           generate-dash: true
 
       - name: Setup Java JDK
-      - uses: actions/setup-java@v4
+        uses: actions/setup-java@v4
         with:
           distribution: 'temurin'
           java-version: '11.0.19'

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -57,7 +57,7 @@ jobs:
           python3 -m pip install tox-gh-actions
 
       - name: Run Linters
-        uses: pre-commit/action@v3.0.0
+        uses: pre-commit/action@v3.0.1
 
       - name: Run the databroker binary
         run: |

--- a/.github/workflows/codeql-analysis.yml
+++ b/.github/workflows/codeql-analysis.yml
@@ -56,7 +56,7 @@ jobs:
 
     # Initializes the CodeQL tools for scanning.
     - name: Initialize CodeQL
-      uses: github/codeql-action/init@v2
+      uses: github/codeql-action/init@v3
       with:
         languages: ${{ matrix.language }}
         # If you wish to specify custom queries, you can do so here or in a config file.
@@ -67,7 +67,7 @@ jobs:
     # Autobuild attempts to build any compiled languages  (C/C++, C#, or Java).
     # If this step fails, then you should remove it and run the build manually (see below)
     - name: Autobuild
-      uses: github/codeql-action/autobuild@v2
+      uses: github/codeql-action/autobuild@v3
 
     # ℹ️ Command-line programs to run using the OS shell.
     # 📚 https://git.io/JvXDl
@@ -81,4 +81,4 @@ jobs:
     #   make release
 
     - name: Perform CodeQL Analysis
-      uses: github/codeql-action/analyze@v2
+      uses: github/codeql-action/analyze@v3

--- a/.github/workflows/project-creation.yaml
+++ b/.github/workflows/project-creation.yaml
@@ -92,7 +92,8 @@ jobs:
             CREATE_ARGS="-e ${{ matrix.example }}"
           fi
           velocitas create -c vapp-core-python $CREATE_ARGS
-          # Make sure that we do not use old shit, we do not want circular dependency to old code
+          # Velocitas CLI fetches an older merged/released version of the example
+          # But we want to check the one in this repository so lets copy it
           if [ "${{ matrix.example }}" != "no-example" ]; then
             cp -r ../sdk/examples/${{ matrix.example }}/* ../app/app
           fi

--- a/.github/workflows/project-creation.yaml
+++ b/.github/workflows/project-creation.yaml
@@ -27,6 +27,9 @@ on:
   pull_request:
     branches:
       - main
+  # Scheduled build to fetch regressions, if any
+  schedule:
+    - cron: "0 4 * * *"
 
 jobs:
   check-project-creation:
@@ -89,6 +92,10 @@ jobs:
             CREATE_ARGS="-e ${{ matrix.example }}"
           fi
           velocitas create -c vapp-core-python $CREATE_ARGS
+          # Make sure that we do not use old shit, we do not want circular dependency to old code
+          if [ "${{ matrix.example }}" != "no-example" ]; then
+            cp -r ../sdk/examples/${{ matrix.example }}/* ../app/app
+          fi
 
       - name: Install dependencies
         shell: bash

--- a/.project-creation/.skeleton/Dockerfile
+++ b/.project-creation/.skeleton/Dockerfile
@@ -32,7 +32,7 @@ RUN /bin/bash -c 'set -ex && \
     pip3 install --no-cache-dir scons; \
     fi'
 
-RUN pip3 install --no-cache-dir pyinstaller==5.9.0 \
+RUN pip3 install --no-cache-dir pyinstaller==6.9 \
     && pip3 install --no-cache-dir patchelf==0.17.0.0 \
     && pip3 install --no-cache-dir staticx \
     && pip3 install --no-cache-dir -r ./workspace/app/requirements.txt \

--- a/NOTICE-3RD-PARTY-CONTENT.md
+++ b/NOTICE-3RD-PARTY-CONTENT.md
@@ -77,7 +77,7 @@
 |actions/upload-artifact|v4|MIT License|
 |danielpalme/ReportGenerator-GitHub-Action|5.2.0|Apache License 2.0|
 |devcontainers/ci|v0.3|MIT License|
-|github/codeql-action|v2|MIT License|
+|github/codeql-action|v3|MIT License|
 |mikepenz/action-junit-report|v4|Apache License 2.0|
-|pre-commit/action|v3.0.0|MIT License|
+|pre-commit/action|v3.0.1|MIT License|
 |softprops/action-gh-release|v1|MIT License|

--- a/examples/seat-adjuster/Dockerfile
+++ b/examples/seat-adjuster/Dockerfile
@@ -44,7 +44,7 @@ RUN velocitas init -v
 
 WORKDIR /workspace/app
 
-RUN pyinstaller --clean -F -s -p src src/main.py
+RUN pyinstaller --hiddenimport=pkg_resources.extern --clean -F -s -p src src/main.py
 
 WORKDIR /workspace/app/dist
 

--- a/examples/seat-adjuster/Dockerfile
+++ b/examples/seat-adjuster/Dockerfile
@@ -32,7 +32,7 @@ RUN /bin/bash -c 'set -ex && \
     pip3 install --no-cache-dir scons; \
     fi'
 
-RUN pip3 install --no-cache-dir pyinstaller==5.9.0 \
+RUN pip3 install --no-cache-dir pyinstaller==6.9 \
     && pip3 install --no-cache-dir patchelf==0.17.0.0 \
     && pip3 install --no-cache-dir staticx \
     && pip3 install --no-cache-dir -r ./workspace/app/requirements.txt \
@@ -44,7 +44,7 @@ RUN velocitas init -v
 
 WORKDIR /workspace/app
 
-RUN pyinstaller --hiddenimport=pkg_resources.extern --clean -F -s -p src src/main.py
+RUN pyinstaller --clean -F -s -p src src/main.py
 
 WORKDIR /workspace/app/dist
 

